SEED-platform/seed
# 2319 api ubid pm import

#### Any background context you want to provide?
#### What's this PR do?
Updated endpoint version -> v3;
monkey test new endpoints;
Modified seed tests accordingly;
Added org_id for import espm from EnergyStar to local dataset.

#### How should this be manually tested?
UI/UX
For testing ESPM template_list:
1. Register an EnergyStar portfolio manager account at https://portfoliomanager.energystar.gov/pm/login.html;jsessionid=63D13D1AF5D158B8B739AAA9D1897F30:
<img width="1029" alt="Screen Shot 2020-08-18 at 8 28 45 PM" src="https://user-images.githubusercontent.com/49968203/90585187-7a80f280-e191-11ea-8235-4ac41b2b3e5e.png">

2. Sign in. Under Reporting, click "Create a New Template". As seen in below a custom template names "api_pm_import" is created:
<img width="1017" alt="Screen Shot 2020-08-18 at 8 32 06 PM" src="https://user-images.githubusercontent.com/49968203/90585542-51ad2d00-e192-11ea-92aa-79f9ed1b8adc.png">

3. From SEED, import this template by choosing "Import Portfolio Manager Data":
<img width="660" alt="Screen Shot 2020-08-18 at 8 36 30 PM" src="https://user-images.githubusercontent.com/49968203/90585658-9df86d00-e192-11ea-9fb0-ae3fe60146e2.png">

4. Log in with your EnergyStar portfolio credential, "api_pm_import" template is returned:
<img width="636" alt="Screen Shot 2020-08-18 at 8 38 13 PM" src="https://user-images.githubusercontent.com/49968203/90585763-d8620a00-e192-11ea-9725-462ea63611a0.png">

For testing ESPM report:
1.Hit submit button and expect successful status. Once template is submitted, data is ready for mapping.
<img width="642" alt="Screen Shot 2020-08-19 at 2 14 19 PM" src="https://user-images.githubusercontent.com/49968203/90685055-73f08a80-e226-11ea-8fa5-5cc74bfea132.png">
<img width="1005" alt="Screen Shot 2020-08-19 at 2 16 29 PM" src="https://user-images.githubusercontent.com/49968203/90685175-a5695600-e226-11ea-8d47-55858cd70360.png">

2. Dataset should show all imported records:
<img width="1404" alt="Screen Shot 2020-08-19 at 2 18 12 PM" src="https://user-images.githubusercontent.com/49968203/90685354-e5303d80-e226-11ea-89e2-d16158b00acd.png">

#### What are the relevant tickets?
#2319 

#### Screenshots (if appropriate)
